dashboard/app/models/script_level.rb

```diff
@@ -177,6 +177,14 @@ def path
     build_script_level_path(self)
   end
 
+  def icon
+    if level.icon
+      level.icon
+    elsif assessment
+      'fa-list-ol'
+    end
+  end
```

Comment:
@joshlory FYI for overriding icon for assessments. Looking at this again, I think that this (a) deserves a comment that captures some of the discussion we've had around icons and (b) I probably should reverse the conditionals, i.e.
```
if assessment
  'fa-list-ol'
else
  level.icon
end
```
Even though there probably aren't currently any level types that have an icon, that we would also make an assessment, if some existing type later did provide a default icon, I think we'd still want assessment to override.